NachoSpigot-Server/src/main/java/org/bukkit/craftbukkit/Main.java

```diff
@@ -23,12 +25,12 @@ public static void main(String[] args) {
         System.setProperty("log4j2.formatMsgNoLookups", "true");
 
         try {
-            if(!SystemUtils.isJavaVersionAtLeast(JavaVersion.JAVA_15)) {
-                System.err.println("It seems like you are not using Java 15!");
-                System.out.println("The use of Java 15 is strongly recommended.");
+            if(!SystemUtils.isJavaVersionAtLeast(JavaVersion.JAVA_17)) {
+                NachoLogger.LOGGER.info("It seems like you are not using Java 17!");
+                NachoLogger.LOGGER.info("The use of Java 17 is strongly recommended.");
```

Comment:
The recommended version is Java 11 or higher I believe

Reply:
in the readme is Java 17 also recommended and with that PR i fixed the last known Java 11 > bug

Reply:
@Lucaskyy What do you think?

Reply:
Performance wise, Java 17 is recommended. Some plugins just don't work on that version, sadly.